src/gluonts/nursery/spliced_binned_pareto/gaussian_model.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+import torch
+import torch.nn.functional as F
+import torch.nn as nn
+
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions.normal import Normal
+
+
+class GaussianModel(nn.Module):
```

Comment:
Is this class used anywhere? I don't get what its purpose is, it seems like a suspiciously thin wrapper around pytorch's `Normal`

Reply:
This class is used in the example notebook, to demonstrate a comparison for the user how to use DistributionalTCN on a familiar distribution and not just our Spliced Binned-Pareto. 

Aside from the function to switch device id, it demonstrates a forward with mini-batching updates, and the forward does not require this way of registering the parameters: https://stackoverflow.com/questions/63047762/correct-way-to-register-a-parameter-for-model-in-pytorch.